spec/model/project_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require_relative "spec_helper"
+require "octokit"
```

Comment:
Seems like this one has to go somewhere else, no? It means application code loaded just so won't work.

Reply:
Github module loads it at `lib/github.rb`. But we mock Github module and the app_client in this file. Since it's mocked `Octokit::Client` fails when I run the single test run. Do I mock wrong?

```
      app_client = instance_double(Octokit::Client)
      expect(Github).to receive(:app_client).and_return(app_client)
```

Reply:
No, I was wrong. I mean, it so happens it works without this require, but there are possible lazily load orders where it wouldn't have. It is more right to have it than not to have it.